usr/share/rear/verify/BORG/default/400_check_archive_access.sh

```diff
@@ -0,0 +1,45 @@
+# Query Borg server for repository information
+# and store it to BORGBACKUP_ARCHIVE_CACHE.
+# This should avoid repeatedly querying Borg server, which could be slow.
+
+borg_archive_cache_create && return
+
+LogPrint "Failed to list Borg archive."
+LogPrint "If you decide to continue, ReaR will partition your disks, but most probably will NOT be able to restore your data!"
+Log "Command \"borg list $BORGBACKUP_OPT_REMOTE_PATH ${borg_dst_dev}${BORGBACKUP_REPO}\" returned: "
+borg_list
+
+rear_workflow="rear $WORKFLOW"
+choices[0]="View '$rear_workflow' log file ($RUNTIME_LOGFILE)"
```

Comment:
I spotted something obviously wrong:
```
unset choices
```
is missing here, cf.
usr/share/rear/layout/prepare/default/300_map_disks.sh
wherefrom you copied it ;-)